coderadar-server/coderadar-core/src/test/java/org/wickedsource/coderadar/analyzingjob/rest/AnalyzingJobControllerTest.java

```diff
@@ -14,13 +14,12 @@
 
 import com.github.springtestdbunit.annotation.DatabaseSetup;
 import com.github.springtestdbunit.annotation.ExpectedDatabase;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
 import org.springframework.http.MediaType;
-import org.wickedsource.coderadar.testframework.category.ControllerTest;
 import org.wickedsource.coderadar.testframework.template.ControllerTestTemplate;
 
-@Category(ControllerTest.class)
+@Tag("ControllerTest.class")
```

Comment:
"ControllerTest.class" is a weird tag. 
Perhaps add a public constant `TAG` to the `ControllerTestTemplate` class containing the String `ControllerTest` and use this within the `@Tag` annotation.

Also: can't we just add the tag to the parent class `ControllerTestTemplate` and remove it from the child classes?

Reply:
I have done this